docs/find-options.md

```diff
@@ -367,8 +367,39 @@ will execute following query:
 SELECT * FROM "post" WHERE "currentDate" > NOW()
 ```
 
-> Note: beware with `Raw` operator. It executes pure SQL from supplied expression and should not contain a user input,
- otherwise it will lead to SQL-injection.
+If you want to provide user input, in order to prevent SQL-injection, you can use the native parameters by providing a second argument to the `Raw` function.
+
+```ts
+import {Raw} from "typeorm";
+
+const loadedPosts = await connection.getRepository(Post).find({
+    currentDate: Raw(alias =>`${alias} > ':date'`, { date: "2020-10-06" })
+});
+```
+
+will execute following query:
+
+```sql
+SELECT * FROM "post" WHERE "currentDate" > '2020-10-06'
+```
+
+If you need to provide an user input that is an array, you can unwrap the native using the following syntax:
```

Comment:
We don't need the particle `an` in `an user input` - it's just `user input`.

We could probably borrow from the [`query-builder`](https://github.com/typeorm/typeorm/blob/master/docs/select-query-builder.md#using-parameters-to-escape-data) docs some of the wording, as well.

```suggestion
If you need to provide user input that is an array, you can bind them as a list of values in the SQL statement by using the special expression syntax:
```

Reply:
Fix and commit (doc:) amended